mail-client/scooby/scooby-0.9.9.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="Full featured e-mail client"
+DESCRIPTION="Scooby is an open source full featured BeOS native e-mail client."
+HOMEPAGE="https://sourceforge.net/p/scooby/"
+COPYRIGHT="Atsushi Takamatsu"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="e53f65326329dd5a85fbdee640f4edefc2d043d3"
+SOURCE_URI="https://github.com/HaikuArchives/Scooby/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="29630d6ef1ff51cb463762d9bed23c8ebadac045e023c28b1ce1e8d72469384c"
+SOURCE_FILENAME="$portVersionedName.tar.gz"
+SOURCE_DIR="Scooby-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 !x86 !x86_64"
```

Comment:
The app also crashes for me, on exit.
It looks like it's a SEGMENTATION VIOLATION happenning in libssl or libcrypto.
I guess it would be appropriate to mark x86_gcc2 as untested, i.e. with a "?":
```
ARCHITECTURES="?x86_gcc2 !x86 !x86_64"
```
Then we can merge ;-)
Thanks!